includes/reader-revenue/class-stripe-connection.php

```diff
@@ -1267,6 +1303,24 @@ public static function create_wc_transaction_payload( $customer, $payment ) {
 			'subscribed'         => self::has_customer_opted_in_to_newsletters( $customer ),
 		];
 	}
+
+	/**
+	 * If a WP user is deleted, cancel their Stripe subscriptions.
+	 *
+	 * @param int $user_id User ID.
+	 */
+	public static function cancel_deleted_user_subscriptions( $user_id ) {
```

Comment:
The method itself does not take "user deletion" into account so it shouldn't state that.

Reply:
Fixed in f3c3c6cbd72fed407e81ad6d567e5c0acb3862e9